.github/workflows/checks-generated-definitions.yml

```diff
@@ -0,0 +1,78 @@
+on:
+  pull_request
+
+permissions:
+  contents: read
+
+jobs:
+  check-definitions:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Get latest tag
+        id: get_latest_tag
+        run: echo "LATEST_TAG=$(git describe --tags $(git rev-list --tags --max-count=1))" >> $GITHUB_ENV
+
+      - name: Checkout latest tag
+        run: git checkout ${{ env.LATEST_TAG }}
+
+      - name: Install dependencies (tag)
+        run: yarn install --frozen-lockfile
+
+      - name: Generate all at tag
+        run: yarn generate:all
```

Comment:
Why do you call `generate:all` if we don't compare the generated file but only the definition file?